package.json

```diff
@@ -20,17 +20,17 @@
   "homepage": "https://github.com/cloudbees/blueocean-acceptance-test#readme",
   "devDependencies": {
     "@jenkins-cd/sse-gateway": "0.0.7",
```

Comment:
Hmmm weird coz all 0.0.8 does is make `js-modules` a peer dep. Adding `js-modules` should fix that, but I'd assume you'd get a build error without it, so strange. I'll have a look.